src/formatters/default_formatter.erl

```diff
@@ -509,10 +509,10 @@ lay_no_comments(Node, Ctxt) ->
             Es = lay_clause_expressions(erl_syntax:block_expr_body(Node), Ctxt1, fun lay/2),
             sep([text("begin"), nest(Ctxt1#ctxt.break_indent, Es), text("end")]);
         catch_expr ->
-            {Prec, PrecR} = preop_prec('catch'),
+            {_, PrecR} = preop_prec('catch'),
             D = lay(erl_syntax:catch_expr_body(Node), set_prec(Ctxt, PrecR)),
             D1 = follow(text("catch"), D, Ctxt#ctxt.break_indent),
-            maybe_parentheses(D1, Prec, Ctxt);
+            maybe_parentheses(D1, 0, Ctxt);
```

Comment:
I basically preserved OTP23 and prior value here. Using the one in OTP24 would work, too, but nobody actually complained about the extra parentheses and maintaining a formatter that formats the code consistently is better than maintaining one that produces different output depending on the OTP version with which it was compiled.

Reply:
Once we stop supporting OTP23… we can revisit this.